build.gradle

```diff
@@ -45,24 +45,23 @@ dependencies {
 	compileOnly "com.google.code.findbugs:jsr305:3.0.2"
 
 	//cardinal components
-	modImplementation "com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-base:${project.cardinal_version}" //only non-fabricmc lib we actually require at runtime
-	modCompileOnly "com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-block:${project.cardinal_version}"
-	modCompileOnly "com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-item:${project.cardinal_version}"
-	modCompileOnly "com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-entity:${project.cardinal_version}"
-	modCompileOnly "com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-chunk:${project.cardinal_version}"
-	modCompileOnly "com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-world:${project.cardinal_version}"
-	modCompileOnly "com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-level:${project.cardinal_version}"
-	include "com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-base:${project.cardinal_version}"
+	include modImplementation("com.github.OnyxStudios.Cardinal-Components-API:cardinal-components-base:${project.cardinal_version}") //only non-fabricmc lib we actually require at runtime
```

Comment:
does this get changed to io as well?

Reply:
whoops, yees..

Reply:
fixed in https://github.com/CottonMC/UniversalComponents/commit/b752a6ebe39bba03f825f0b2d088942fdc582853